src/services/rollouts.js

```diff
@@ -124,6 +125,83 @@ module.exports = class RolloutsHelper {
 			throw error
 		}
 	}
+
+	/**
+	 * Rollout details
+	 * @method
+	 * @name details
+	 * @param {String} rolloutId - Rollout id
+	 * @param {String} orgId - Organization id
+	 * @param {String} loggedInUserId - User id
+	 * @returns {JSON} - Rollout Details
+	 */
+	static async details(rolloutId, orgId, loggedInUserId) {
+		try {
+			let result = {
+				organization: {},
+			}
+
+			const rollout = await rolloutQueries.findOne({
+				id: rolloutId,
+				organization_id: orgId,
+				user_id: loggedInUserId,
+				type: common.ROLLOUT_TYPE_PROGRAM,
```

Comment:
@priyanka-TL why this filter?

Reply:
@aks30 removed, only for list its required